argoverse/utils/make_att_files.py

```diff
@@ -0,0 +1,380 @@
+import sys
+import argoverse
+from argoverse.data_loading.argoverse_tracking_loader import ArgoverseTrackingLoader
+import open3d as o3d
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+import json
+import glob
+import scipy
+import scipy.interpolate as interpolate
+import torch.nn.functional as F
+import torch
+import cv2
+import pickle
+
+dict_color = {}
+dict_color["easy"] = (0, 1.0, 0)
```

Comment:
could we add a small comment next to these, with their qualitative color description, e.g. light-green, etc?